src/bin/bat/app.rs

```diff
@@ -82,11 +82,10 @@ impl App {
             Some("always") => PagingMode::Always,
             Some("never") => PagingMode::Never,
             Some("auto") | None => {
-                if self.matches.occurrences_of("plain") > 1 {
+                if self.matches.occurrences_of("plain") > 1
+                    || self.matches.is_present("no-paging") {
```

Comment:
I don't think that's really better. The comment does not apply to `--no-paging`.